scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -361,6 +361,13 @@ class ScalametaParser(input: Input, dialect: Dialect) { parser =>
           } else if (curr.is[RightBracket]) {
             if (!sepRegions.isEmpty && sepRegions.head == RegionBracket) sepRegions.tail
             else sepRegions
+          } else if (curr.is[EOF]) {
```

Comment:
We weren't adding outdents when EOF was met, which can easily happen.

Reply:
I don't think people usually leave EOF, but it failed on a number of places in scala3doc